test/e2e/install_test.go

```diff
@@ -64,4 +68,45 @@ var _ = Describe("Operator Install", func() {
 		err = c.Delete(ctx, operator)
 		Expect(err).ToNot(HaveOccurred())
 	})
+
+	It("errors when Rukpak is not able to install the operator", func() {
```

Comment:
Could we move this test to `controllers/operator_controller_test.go`? We don't actually need rukpak running for this, right? We just need to create BDs with various  status condition types/reason/message combos so that we exercise the operator controller's code paths.

The motivation is to reserve e2e tests for things that really do require the full system running.